.rubocop.yml

```diff
@@ -2,6 +2,7 @@
 # https://github.com/bbatsov/rubocop/blob/master/config/
 
 AllCops:
+  TargetRubyVersion: 2.2.0
```

Comment:
why is this parameter required?
does it mean that it should "=" 2.2? or >= 2.2 ? the gemspec defines it as >= 2.2, and miq is already at a higher version

Reply:
This tag is used to describe the minimum supported version. So it is a '>= 2.2'.

When running rubocop on my local, it uses ruby 2.1 parser instead that the current ruby version (i.e. 2.4.1). Adding this line, it ensures that Rubocop doesn't use a parser for a version below 2.2.0.